Assignment2/eduardogeuens-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,56 @@
+{
+	"@context": {
+		"re": "http://assignment2b.com#",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"Class01": "re:Class01",
+		"Sensor029": "Class01:Sensor029",
+		"Computer101": "Class01:Computer101",
+		"Measurement8401": "Sensor029:Measurement8401",
+		"User10A": "Computer101:User10A",
+		"hasMeasurement": "re:hasMeasurement",
+		"hasTemperature": "re:hasTemperature",
+		"includes": "re:includes",
+		"hasOwner": "re:hasOwner",
+		"hasName": "re:hasName",
+		"atTime":"re:atTime"		
+	},
+	"@id": "http://assignment2b.com/Class01",
+	"@type": "re:Class01",
+	"includes": [
+		{
+			"Sensor029":{
+				"@id": "http://assignment2b.com/Class01/Sensor029",
+				"@type": "Class01:Sensor029",

```

Comment:
This type definition works, but semantically isn't ok. The best way to define it might be defining using your prefix re and the concept name, e.g. "@type": "re:Sensor".
This comment also applies for the other type definitions.